package.json

```diff
@@ -9,10 +9,13 @@
   },
   "devDependencies": {
     "fs-extra": "10.0.0",
-    "markdown-table": "3.0.2"
+    "husky": "7.0.0",
```

Comment:
The latest version of Husky is `7.0.4`, I somehow managed to install the older version...

I'll go fix this now.